python/lsst/afw/image/apCorrMap.py

```diff
@@ -0,0 +1,7 @@
+from __future__ import absolute_import, division, print_function
+
+from ._apCorrMap import ApCorrMap
+
+ApCorrMap.values = lambda self: [self[name] for name in self.keys()]
+ApCorrMap.items = lambda self: [(name, self[name]) for name in self.keys()]
+ApCorrMap.__iter__ = lambda self: iter(self.keys())
```

Comment:
Don't forget the `del _values` either. Yes it is really ugly syntax...
At some point we may want to investigate writing some magic for it instead. Unfortunately `continue class` was not accepted upstream :( Anyway, I suspect that most cases can be pushed up to the C++ wrapper layer. But lets not do that now.